packages/terra-switch/translations/sv-SE.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@ryanthemanuel did we want to stop adding sv-SE translations or wait for the react-intl bump?